packages/react-sdk/src/locales/de/neoboard.json

```diff
@@ -156,6 +156,7 @@
     "deleteElement_other": "Elemente löschen",
     "duplicateActiveElement_one": "Element duplizieren",
     "duplicateActiveElement_other": "Elemente duplizieren",
+    "fontForm": "Schriftstil wählen",
```

Comment:
A „Stil“ would be bold or italic to me. I would say it should be named „Schriftart“ instead. This is quite common

Reply:
Done